include/sot/torque_control/torque-offset-estimator.hh

```diff
@@ -79,7 +79,6 @@ namespace dynamicgraph {
 
         DECLARE_SIGNAL_IN(base6d_encoders,          dynamicgraph::Vector);
         DECLARE_SIGNAL_IN(accelerometer,            dynamicgraph::Vector);
-        DECLARE_SIGNAL_IN(gyroscope,                dynamicgraph::Vector);
```

Comment:
Maybe terminating is a bit too much. Just print a message to warn the user that the gyroscope is measuring a significant angular velocity, which might jeopardize the calibration procedure.